app/frontend/src/store/projects.ts

```diff
@@ -15,17 +17,92 @@ export const $selectedProject = createStore<Project | null>(null);
 /**
  * Event to be called when selected project state should be updated
  */
-export const projectSelected = createEvent<Project | null>();
+export const projectSelected = createEvent<Project | null | undefined>();
```

Comment:
pls describe the use cases of `null` and `undefined` in jsdoc. What is the difference?